manageiq-operator/deploy/crds/manageiq_v1alpha1_manageiq_cr.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: manageiq.example.com/v1alpha1
+kind: Manageiq
+metadata:
+  name: miq
+spec:
+  # Add fields here
+  appName:  "manageiq"
+  applicationAdminPassword: "smartvm" 
+  applicationDomain: "miqproject.apps-crc.testing"
```

Comment:
Is this an example file? I would expect that this would always need to be changed so does it make sense to provide a default?

Reply:
how about this ` applicationDomain: # e.g. "miqproject.apps-crc.testing"`

Reply:
Sure? I would prefer to use an actual documentation hostname though. `manageiq.example.com` maybe?